### ISSUE TITLE
Create a Konami 005885 device to unify implementations in various drivers

### ISSUE THREAD

**Issue by Osso13** [mamedev/mame#2053]
Single, not shared implementations are found in:

1. finalizr.cpp
2. ironhors.cpp
3. ddribble.cpp
4. jackal.cpp

**Comment by FluBBaOfWard** [comment]
Yes please. The current implementations differs in a number of ways though they shouldn't.
It has 64 sprites (no more, no less), Iron Horse has missing sprites in MAME because the driver specifies to small sprite ram (256 bytes when it should be 320 bytes).
Sprite sizes are 32x32, 16x16, 16x8, 8x16 & 8x8.
The only difference between the games are how the tile code is calculated (and Jackal using 256 color bg tiles).
Interrupts should also be handled in a more common way (same as 005849).
I have all 4 games running on a Nintendo DS with the same graphics driver.
